src/main/java/core/basesyntax/service/ConvertDataFromFile.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ConvertDataFromFile {
```

Comment:
actually this service doesn't work with files, it takes a list of strings as input so let's rename this interface properly